src/redux/remoteData/asyncActionCreator.ts

```diff
@@ -143,3 +145,16 @@ export const startup = () => {
     dispatch(fetchCallCount());
   };
 };
+
+const migrateLegacyCompletedIssues = (dispatch: Dispatch<ApplicationState>) => {
+  const LEGACY_COMPLETED_ISSUES_KEY = 'org.5calls.completed';
+  const legacyCompletedIssues = localStorage.getItem(LEGACY_COMPLETED_ISSUES_KEY);
+  if (legacyCompletedIssues) {
+    const ids = JSON.parse(legacyCompletedIssues);
+    ids.forEach((id) => {
+      dispatch(selectIssueActionCreator(id));
```

Comment:
These need to be selected before completed? Does that cause a bunch of UI jumpiness on the migration?

Reply:
Fixed that in commit 105120e